action_server/poetry.lock

```diff
@@ -1173,123 +1189,126 @@ name = "robocorp-tasks"
 version = "2.6.0"
 description = "The automation framework for Python"
 optional = false
-python-versions = ">=3.9,<4.0"
-files = [
-    {file = "robocorp_tasks-2.6.0-py3-none-any.whl", hash = "sha256:df5776c5da38f794b576d42d6a0a9dad86280bfbe1b99982d136f8963aca70ec"},
-    {file = "robocorp_tasks-2.6.0.tar.gz", hash = "sha256:295936fa8e3344e4d83e7a28481e1193dc4af7aea8ff1af964920525221d8284"},
-]
+python-versions = "^3.9"
+files = []
+develop = true
 
 [package.dependencies]
-docstring_parser_fork = ">=0.0.5,<0.0.6"
-psutil = ">=5.9,<6.0"
+docstring_parser_fork = "^0.0.5"
+psutil = "^5.9"
+pydantic = "^2.5.2"
 robocorp-log = ">=2.4,<3"
 
+[package.source]
+type = "directory"
```

Comment:
This lock file should probably not be added (the lock needs to be done based on the pypi deps, not on the devinstall deps).